internal/resources/fetcher.go

```diff
@@ -2,16 +2,23 @@ package resources
 
 import (
 	"context"
+	"fmt"
+	"net/url"
 
 	policiesv1 "github.com/kubewarden/kubewarden-controller/pkg/apis/policies/v1"
+	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/client-go/dynamic"
 	ctrl "sigs.k8s.io/controller-runtime"
 )
 
+const kubewardenNamespace = "kubewarden"
```

Comment:
shouldn't this be parametrized at some point? ('m thinking it's part of this issue in the epic).

For finding the policy-server service and kubewarden namespace, we need to pass this to the executable IIUC.